packages/js/product-editor/src/components/attribute-empty-state/attribute-empty-state.tsx

```diff
@@ -19,7 +18,7 @@ type AttributeEmptyStateProps = {
 };
 
 export const AttributeEmptyState: React.FC< AttributeEmptyStateProps > = ( {
```

Comment:
Any reason we need to keep this component if it's no longer in use?

Reply:
Nope, just removed in https://github.com/woocommerce/woocommerce/pull/38051/commits/9048f92843b922d9741477152645f4dc07118753 👍🏻